Xamarin.Forms.Platform.Android/Renderers/SwipeViewRenderer.cs

```diff
@@ -53,7 +53,8 @@ public class SwipeViewRenderer : ViewRenderer<SwipeView, AView>
 
 		public SwipeViewRenderer(Context context) : base(context)
 		{
-			SwipeView.VerifySwipeViewFlagEnabled(nameof(SwipeViewRenderer));
+			if (ShouldVerifySwipeViewFlagEnabled())
```

Comment:
This check doesn't work because _context is assigned after this statement

`ShouldVerifySwipeViewFlagEnabled` should just be changed to a static method that takes the context as an input

Reply:
Good catch!. Changed.